src/HomeWork_12/Documents.java

```diff
@@ -0,0 +1,61 @@
+package HomeWork_12;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Documents {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Set<String> documentNumbers = new HashSet<>();  // коллекция для уникальных номеров документов
+        Set<String> processedDocuments = new HashSet<>();  // коллекция для хранения уже проверенных документов
```

Comment:
We don't need second collection. Both of them will be equals, numbers are unique